test/create-write-stream.js

```diff
@@ -30,7 +30,7 @@ test('createWriteStream(key) with options', async function (t) {
   t.alike(await drive.entry('/new-script.sh'), {
     key: '/new-script.sh',
     value: {
-      executable: true,
+      executable: !isWin,
```

Comment:
I decided to not skip the entire test in this case, so at least createWriteStream() with options is run anyway.